editor/editor_node.cpp

```diff
@@ -2877,9 +2877,12 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 
 			if (!scene->get_scene_file_path().is_empty()) {
 				String path = scene->get_scene_file_path();
+				String root_name = EditorNode::adjust_scene_name_casing(scene->get_name());
+				String ext = path.get_extension().to_lower();
+				path = path.get_base_dir() + "/" + root_name + "." + ext;
```

Comment:
I'm not sure if `get_base_dir()` can return a trailing `/` but using `plus_file()` should ensure there will only be one slash and not two.
```suggestion
				path = path.get_base_dir().plus_file(root_name + "." + ext);
```

Reply:
Seems like it can't find `plus_file` in String

Reply:
Is it `path_join` ?

Reply:
Yes.